### DISCUSSION TITLE
How to unsubscribe one sub in two codes.

### DISCUSSION THREAD

**Discussion by Deku660** [pocketbase/pocketbase#4890] [Q&A]
Example.
A. collection('example').subscribe('*')  -> dont have filter.
B. collection('example').subscribe('*', filter:~) -> have filter.

In this case, how to unSubscribe one code?
i want to stop one code.
collection('example').unsubscribe(A)

**Comment by Deku660** [comment]
Im sorry... my eng is very bad

**Reply by ganigeorgiev** [reply]
Please don't open multiple posts for the same thing without actually providing any new information.

As mentioned in your [other post](https://github.com/pocketbase/pocketbase/discussions/4889), the only way to unsubscribe from a specific subscription is to call its returned unsubscribe func:

```js
const unsub1 = await pb.collection("example").subscribe(...)

const unsub2 = await pb.collection("example").subscribe(...)

// unsubscribe just the first subscription
unsub1();
```

I don't know how to make it more clear than this.

**Reply by Deku660** [reply]
Thanks !!!!
i wanted this!
